src/libs/SidebarUtils.ts

```diff
@@ -388,8 +388,9 @@ function getOptionData(
                     ? Localize.translate(preferredLocale, 'workspace.invite.invited')
                     : Localize.translate(preferredLocale, 'workspace.invite.removed');
             const users = Localize.translate(preferredLocale, targetAccountIDs.length > 1 ? 'workspace.invite.users' : 'workspace.invite.user');
-            result.alternateText = `${verb} ${targetAccountIDs.length} ${users}`;
-
+            const actorDisplayName = ReportUtils.getDisplayNameForParticipant(lastAction.actorAccountID);
+            result.alternateText =
+                lastAction.actorAccountID === currentUserAccountID ? `${verb} ${targetAccountIDs.length} ${users}` : `${actorDisplayName} ${verb} ${targetAccountIDs.length} ${users}`;
```

Comment:
remove `const actorDisplayName` and replace `currentUserAccountID` with `[lastActorDisplayName](https://github.com/koko57/App/blob/cba06080f5501038a8b9ef3fa156c532054af3e5/src/libs/SidebarUtils.ts#L346)`

Reply:
I gotchu, sent a PR [here](https://github.com/Expensify/App/pull/35527).